Model/bin/ssCommitSuggesterIndex

```diff
@@ -0,0 +1,13 @@
+#/bin/bash
+
+SOLR_URL=$1
+
+if [ -z "$SOLR_URL" ]; then
+  echo ""
```

Comment:
please add a sentence or two in the usage describing what this script does, and when it might be called.

Reply:
Done